### DISCUSSION TITLE
Projection output ISO XML 20022

### DISCUSSION THREAD

**Discussion by SergioGuirreri** [finos/common-domain-model#3415] [Q&A] [answered]
Hello everyone,

I am writing to request your support regarding the translation of CDM reportable objects in the ISO XML 20022 schema. I would like to know if this is something that will be addressed in the upcoming weeks/months.

If a testpack or documentation for this already exists, could you please provide me with more information on where I can find it?

Thank you for your assistance.
Sergio

**Comment by tomhealey-icma** [comment, marked as answer]
Thank you for your question. This is on the roadmap and initial research has begun but no commitments have been made.
